src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
+
+    Boolean delete(Long id);
```

Comment:
Consider using the primitive `boolean` instead of `Boolean` for the `delete` method's return type to avoid unnecessary boxing and potential nullability issues.